src/rasterizer/model.py

```diff
@@ -0,0 +1,50 @@
+import torch
+import time
+from rasterizer import Rasterizer
+
+# N is num of glyphs; D_in is input dimension;
+# H is hidden dimension; D_out is output dimension.
+N, D_in, H, D_out = 1, 1000, 600, 256
+
+# Create random Tensors to hold inputs and outputs
+x = torch.randn(N, D_in)
+y = torch.randn(N, D_out)
+
+# Use the nn package to define our model as a sequence of layers
+model = torch.nn.Sequential(
+    torch.nn.Linear(D_in, H),
+    torch.nn.ReLU(),
+    torch.nn.Linear(H, H),
+    torch.nn.ReLU(),
+    # to test with raster change H on line 21 to 256 and uncomment line below
+    Rasterizer(resolution=256,
+    steps=128,
+    sigma=1e-2,
+    method='base_half'),
+    torch.nn.Linear(H, D_out),
+    torch.nn.ReLU(),
+)
+
+loss_fn = torch.nn.MSELoss(reduction='sum')
+
+learning_rate = 1e-4
+# t = number of back prop
+for t in range(1):
+    # start time for effeciency of model
+    start = time.time()
+
+    y_pred = model(x)
+    end = time.time()
+    # duration model ran in seconds
+    print(end - start)
```

Comment:
This currently times how long the _whole model_ is taking to forward propagate and back propagate. Really what we'd like to see is a layer-by-layer breakdown of how long each layer is taking to forwardprop and backprop. Idk how you would do that, though 😛

Reply:
do we really care about the other dummy layers?

Reply:
We care about the speed of the rasterizer as compared to "reasonably sized" dummy generators and discriminators - and we can only know what a "reasonable size" is when we time it.

Also reminded me of a quote:
> If you use a ruler to measure a table, you may also be using the table to measure the ruler.

Reply:
You don't need a complicated layer by layer break down. Simply time the generator, then time the discriminator (with say a random input) and sum them